src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -29,35 +29,89 @@ class DialogmoteService(
     private val oppfolgingstilfelleClient: OppfolgingstilfelleClient,
     private val pdfGenClient: PdfGenClient,
 ) {
-    fun getDialogmote(
-        moteUUID: UUID
+    suspend fun getDialogmote(
+        moteUUID: UUID,
+        callId: String,
     ): Dialogmote {
         return database.getDialogmote(moteUUID).first().let { pDialogmote ->
-            extendDialogmoteRelations(pDialogmote)
+            dialogmote(pDialogmote, callId)
+            extendDialogmoteRelations(pDialogmote, callId)
+        }
+    }
+
+    suspend fun getDialogmote(
+        id: Int,
+        callId: String,
+    ): Dialogmote {
+        return database.getDialogmote(id).first().let { pDialogmote ->
```

Comment:
Det ser ikke ut som det er behov for noen av disse endringene her  i `DialogmoteService` og i `DialogmotedeltakerService`. Henting av nærmeste leder her er overflødig og da trengs det ingen endringer knyttet til oppretting av objekt for dialogmote som hentes.